src/ceph_osd.cc

```diff
@@ -247,6 +248,8 @@ int main(int argc, const char **argv)
     forker.exit(0);
   }
 
+  ceph_init_huge_page_pools(g_conf);
```

Comment:
This will be needed for ceph-bluestore-tool and ceph-objectstore-tool too.  We could put it in ceph_context.cc, but then we'd eat up huge pages for non-OSDs too.  Perhaps it should go in BlueFS mount() and/or mkfs(), since that's the only user.

Reply:
Hmm, if we make the pool allocate new pages on demand, up to the max, instead of preallocating them, then doing it in CephContext would be free if they're not used.